docs/contrib/Logger.rst

```diff
@@ -317,7 +324,10 @@ call the utl::error/warn with a Tool ID and message ID. For
 compatibility these are defaulted to 'UKN' and '0000' until they are
 updated.
 
-Do no use "puts", ie, print) errors in regression tests. The logger prints the error now.
+Do no use "puts", i.e, (print) errors in regression tests. Use the report
```

Comment:
I don' really agree with this statement.
There is really no reason regression tests cannot use 'puts', especially as an alternative to the Logger::report.
It is an essential part of tcl.